con_knative-OCP-4x.adoc

```diff
@@ -0,0 +1,24 @@
+// This assembly is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+
+= Knative on an OpenShift 4.0 cluster
+
+== Developer Preview 
+
+NOTE: The functionality introduced by Knative on an OpenShift cluster is preview only. Red Hat support is not provided, and this release should not be used in a production environment.
+
+You will need cluster administrator privileges to install and use Knative on an OpenShift cluster.
+
+=== Supported platform version
+
+[cols="50,50"]
+|===
+|** Platform**     | **Supported versions**   
+| OpenShift    | 4.0 [Developer Preview]     
+|===
+
+
+.Additional resources
+For more information on gettng started, visit link:https://try.openshift.com/[try.openshift.com].

```

Comment:
Needs to be called out that this try.openshift.com is now the dev preview 4.0 cluster is accessed and the only way we support it. Not an additional resource, this should be highlighted as how you get a 4.0 cluster (prerequisite)